installers/community/arm64/edge/start.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Copyright 2017 Konrad Zapalowicz <bergo.torino@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Start EdgeX Foundry services in right order, as described:
+# https://wiki.edgexfoundry.org/display/FA/Get+EdgeX+Foundry+-+Users
+
+COMPOSE_FILE=${1:-docker-compose.yml}
+echo "Using compose file: $COMPOSE_FILE"
+
+
+install_cors_plugin () {
+    echo $'\n'
+    echo  "Installing cors plugin"
+    curl -X POST http://localhost:8001/plugins/ \
+    --data "name=cors"  \
+    --data "config.origins=*" \
+    --data "config.methods=GET" \
+    --data "config.methods=POST" \
+    --data "config.methods=OPTIONS" \
+    --data "config.headers=Accept" \
+    --data "config.headers=Accept-Version" \
+    --data "config.headers=Content-Length" \
+    --data "config.headers=Content-MD5" \
+    --data "config.headers=Content-Type" \
+    --data "config.headers=Date" \
+    --data "config.headers=X-Auth-Token" \
+    --data "config.headers=Authorization" \
+    --data "config.exposed_headers=X-Auth-Token" \
+    --data "config.credentials=true" \
+    --data "config.max_age=3600"
+}
```

Comment:
Is `*` an appropriate origin for all the places where this will be run and accessed?

Some indenting might make the continuation of the command more clear.

```suggestion
    curl -X POST http://localhost:8001/plugins/ \
      --data "name=cors"  \
      --data "config.origins=*" \
      --data "config.methods=GET" \
      --data "config.methods=POST" \
      --data "config.methods=OPTIONS" \
      --data "config.headers=Accept" \
      --data "config.headers=Accept-Version" \
      --data "config.headers=Content-Length" \
      --data "config.headers=Content-MD5" \
      --data "config.headers=Content-Type" \
      --data "config.headers=Date" \
      --data "config.headers=X-Auth-Token" \
      --data "config.headers=Authorization" \
      --data "config.exposed_headers=X-Auth-Token" \
      --data "config.credentials=true" \
      --data "config.max_age=3600"
}
```

Reply:
Commented out this code as it is not in use right now. But noted the *, it should be parameterized at the very least.